support/dev-tools/polkadot-utils/src/settings.ts

```diff
@@ -0,0 +1,8 @@
+export default {
+    nodeAddress: "ws://127.0.0.1:9944",
+    types: {
+        FractalId: "u64",
+        MerkleTree: "Raw"
+    },
```

Comment:
Are you able to include the JSON file at blockchain/types.json? That will be the source of truth for our types.